perl-package/AI-MXNet/lib/AI/MXNet/Logging.pm

```diff
@@ -17,8 +17,11 @@
 
 package AI::MXNet::Logging;
 ## TODO
+use strict;
+use warnings;
 use Mouse;
-sub warning { shift; warn sprintf(shift, @_) . "\n" };
+$AI::MXNet::Logging::silent = 0;
```

Comment:
`our $silent = 0;` would achieve the same thing here, and then uses of the variable in this package wouldn't need to be fully qualified.

Reply:
ok, I'll change, thanks.